README.md

```diff
@@ -38,6 +43,21 @@ $ SPLK_SERVICE_NAME=my-python-app \
     splk-py-trace python main.py --port=8000
 ```
 
+Notes:
+- Depending on the default python version on your system, you might want to use `pip3` and `python3` instead. 
+- To be able to run `splk-py-trace` and `splk-py-trace-bootstrap`, the directory pip installs scripts to will
+  have to be on your system's PATH environment variable. Generally, this works out of the box when using
+  virtual environments, installing packages system-wide or in container images. In some cases, pip may install
+  packages into your user local environment. In that case you'll need to add your Python user base's bin directory
+  to your system path. You can find out your Python user base as follows by running `python -m site --user-base`.
+
+  For example, if `python -m site --user-base` reports that `/Users/splunk/.local` as the Python user base, then
+  you can add the directory to your path on Unix like system as follows:
+
+  ```
+  export PATH="/Users/splunk/.local/bin:$PATH"
+  ```
```

Comment:
can we move this to the requirements section?

Reply:
I don't think these belong in requirements. They're helpful generic python tips and not very specific to this library.